api/envoy/extensions/filters/http/aws_request_signing/v3/aws_request_signing.proto

```diff
@@ -48,4 +51,7 @@ message AwsRequestSigning {
   // to calculate the payload hash. Not all services support this option. See the `S3
   // <https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-header-based-auth.html>`_ policy for details.
   bool use_unsigned_payload = 4;
+
+  // A given header that matches any of the configured matchers will not be signed.
```

Comment:
Let's rephrase this.  I propose:

```
A list of request headers that will be excluded from signing.
```
You might expand a bit on the matcher implementation (haven't gotten that far yet) on whether these are prefix matchers etc.

Reply:
expanded and added an example.